lib/common.js

```diff
@@ -143,21 +159,12 @@ const restoreOverriddenRequests = function() {
  * @return {string}
  */
 function stringifyRequest(options, body) {
-  const { method = 'GET', path = '' } = options
-  let { port } = options
-
-  if (
-    (options.proto === 'https' && (port === 443 || port === '443')) ||
-    (options.proto === 'http' && (port === 80 || port === '80'))
-  ) {
-    port = ''
-  } else {
-    port = `:${port}`
-  }
+  const { method = 'GET', path = '', port } = options
+  const portStr = addNonStandardPort(options.proto, port) ? `:${port}` : ''
```

Comment:
What do you think about moving the comma into the return value? I think it would clean these up a bit.

Reply:
The colon? 
Change the new function to always return a string? Either an empty string for the "false" case or `:${port}`. 

Another option would be to require the host and return the host string, with the port if non standard.

Reply:
Ah, yes, the colon 😝